_data/people/matthewfeickert.yml

```diff
@@ -67,3 +67,10 @@ presentations:
     meetingurl: https://indico.cern.ch/event/894127/
     location: Princeton, U.S.A.
     focus-area: as
+  - title: "pyhf: A Pure Python Statistical Fitting Library with Tensors and Autograd"
+    date: 2020-02-27
+    url: https://indico.cern.ch/event/897086/attachments/2001484/3341223/Feickert_pyhf-NSF-Review-IRIS-HEP_2020-02-27.pdf
```

Comment:
Can this file be renamed? ;)

Reply:
Actually both files...

Reply:
> Can this file be renamed?

Uh, I guess? Can you explain what the problem is so I have an idea of what it can be renamed to? There aren't any name collisions with other presentations, so I'm a bit confused.

> Actually both files...

What is the other file you mean?

Reply:
Oh. I think I understand now. I think this is fine given the email conversation that I had with @rct225 and @pelmer. Though I'll defer to them. If the files names need to be changed then @rct225 will need to change them.

Reply:
@henryiii It's fine.  This was discussed between me, @pelmer and @matthewfeickert.

Reply:
Okay, thanks!

Reply:
Thanks @henryiii, thanks @rct225! I wasn't expecting either of you to be so responsive tonight, but I appreciate it. :)